src/app/models/book-borrow-data-model.ts

```diff
@@ -0,0 +1,17 @@
+import { returnBookOption } from '../return-book-type.enum';
+
+export class BookBorrowData {
```

Comment:
What do you think about different name? Ending class name with 'data' is wrong approach. For example you can have 'UserData', 'BookData', 'OrderData' everything can be 'Data' 😆 It this case data suffix doesn't say anything